### PR TITLE
SLING-11035 update to bundle-parent 46

### PR DESCRIPTION
**Pull request by joerghoh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +80,14 @@
             <groupId>org.osgi</groupId>
             <artifactId>osgi.core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.versioning</artifactId>
```
Comment (kwin) [inline pom.xml:86]: we should always explicitly mention the scope "provided"

File: pom.xml [hunk 2]
```diff
@@ -82,6 +80,14 @@
             <groupId>org.osgi</groupId>
             <artifactId>osgi.core</artifactId>
```
Comment (kwin) [inline pom.xml]: we should switch to individual OSGi artifacts

File: pom.xml (hunk 1 above)
Comment (joerghoh) [reply pom.xml:86]: done

File: pom.xml (hunk 2 above)
Comment (joerghoh) [reply pom.xml]: done

**Pull request merged**